bin/cml-tensorboard-dev.test.js

```diff
@@ -1,10 +1,16 @@
 jest.setTimeout(200000);
 
-const fs = require('fs');
-const { exec } = require('../src/utils');
-const credentials =
+const { exec, is_proc_running, sleep } = require('../src/utils');
+const CREDENTIALS =
   '{"refresh_token": "1//03FiVnGk2xhnNCgYIARAAGAMSNwF-L9IrPH8FOOVWEYUihFDToqxyLArxfnbKFmxEfhzys_KYVVzBisYlAy225w4HaX3ais5TV_Q", "token_uri": "https://oauth2.googleapis.com/token", "client_id": "373649185512-8v619h5kft38l4456nm2dj4ubeqsrvh6.apps.googleusercontent.com", "client_secret": "pOyAuU2yq2arsM98Bw5hwYtr", "scopes": ["openid", "https://www.googleapis.com/auth/userinfo.email"], "type": "authorized_user"}';
 
+const is_tb_running = async () => {
+  await sleep(2);
```

Comment:
sleep() is a potential race condition. is there a way to avoid this sleep?

Reply:
Its just in the test to be sure that the process is in memory after a period of time. Its safe.

Reply:
Time-based synchronization is never safe. But it might be ok to keep it in tests.